electricitymap/contrib/config/model.py

```diff
@@ -104,7 +104,7 @@ class Zone(StrictBaseModelWithAlias):
     disclaimer: Optional[str]
     flag_file_name: Optional[str]
     parsers: Parsers = Parsers()
-    sub_zone_names: Optional[List[str]] = Field(None, alias="subZoneNames")
+    sub_zone_names: Optional[List[ZoneKey]] = Field(None, alias="bypassedSubZones")
```

Comment:
copy paste gone wrong?

Reply:
No it's intended, since we have a ZoneKey type alias, I might as well use it.

Reply:
Oups!

Reply:
Fixed